jenkinsfiles/README.md

```diff
@@ -1,17 +1,19 @@
 ## Mobile Application configurations
 
-Folder contains Jenkins pipeline definitions for mobile app builds. 
+This folder contains Jenkins pipeline definitions for mobile app builds.
+
 Please refer to specific platform and architecture for more information.
 
-## Jenkinsfiles
+## `jenkinsfile`s
 
-Files have groovy extension to support syntax higlighing.
-When moving to repositories they should be renamed to `Jenkinsfile`.
+Files have groovy extension to support syntax highlighting.
+When moving to repositories they should be renamed to `jenkinsfile`.
 
-- android_jenkinsfile.groovy
-- cordova_jenkinsfile.groovy
+- `android_jenkinsfile.groovy`
+- `cordova_jenkinsfile.groovy`
 
 ## Development
 
-Pipeline files can be modifed for your own needs as long they would launch on the kubernetes node (docker slave).
-Please follow general Jenkins pipelines documentation here: https://jenkins.io/doc/book/pipeline/jenkinsfile/
+Pipeline files can be modified for your own needs as long they would launch on the Kubernetes node (Docker slave).
```

Comment:
I wrote this but its wrong. We should say: as long as commands/ plugins would lainch..